python/nav/eventengine/alerts.py

```diff
@@ -108,6 +113,8 @@ def make_alert_history(self):
             start_time=self.time,
             end_time=INFINITY if self.state == Event.STATE_START else None,
         )
+        if self.severity_rules:
+            self.severity = self.severity_rules.evaluate(self)
```

Comment:
See previous comment

Reply:
Because the AlertGenerator is mutable, so it doesn't necessarily have all the information needed to evaluate a severity rule until the last moment: When the model object is actually being created for insertion into the database.